### PR TITLE
more cleanup

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: views/default/admin/users/add.php [hunk 1]
```diff
@@ -3,4 +3,4 @@
  * Display an add user form.
  */
 
-echo elgg_view_form('useradd', ['prevent_double_submit' => true]);
+echo elgg_view_form('useradd', []);
```
Comment (jeabakker) [inline views/default/admin/users/add.php]: why leave the empty array?

File: views/default/admin/users/admins.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-$make_admin_form = elgg_view_form('admin/user/makeadmin', ['prevent_double_submit' => true]);
+$make_admin_form = elgg_view_form('admin/user/makeadmin', []);
```
Comment (jeabakker) [inline views/default/admin/users/admins.php]: why leave the empty array?

File: mod/invitefriends/views/default/resources/friends/invite.php [hunk 3]
```diff
@@ -13,6 +13,6 @@
 }
 
 echo elgg_view_page(elgg_echo('friends:invite'), [
-	'content' => elgg_view_form('friends/invite', ['prevent_double_submit' => true]),
+	'content' => elgg_view_form('friends/invite', []),
```
Comment (jeabakker) [inline mod/invitefriends/views/default/resources/friends/invite.php]: why leave the empty array?

File: views/default/admin/configure_utilities/maintenance.php [hunk 4]
```diff
@@ -3,4 +3,4 @@
  * Maintenance mode settings
  */
 
-echo elgg_view_form('admin/site/set_maintenance_mode', ['prevent_double_submit' => true]);
+echo elgg_view_form('admin/site/set_maintenance_mode', []);
```
Comment (jeabakker) [inline views/default/admin/configure_utilities/maintenance.php]: why leave the empty array?

File: views/default/admin/configure_utilities/robots.php [hunk 5]
```diff
@@ -12,4 +12,4 @@
 	echo elgg_view_message('warning', elgg_echo('admin:robots.txt:physical'));
 }
 
-echo elgg_view_form('admin/site/set_robots', ['prevent_double_submit' => true]);
+echo elgg_view_form('admin/site/set_robots', []);
```
Comment (jeabakker) [inline views/default/admin/configure_utilities/robots.php]: why leave the empty array?

File: views/default/admin/security.php [hunk 6]
```diff
@@ -8,4 +8,4 @@
 	'value' => elgg_echo('admin:security:settings:description'),
 ]);
 
-echo elgg_view_form('admin/security/settings', ['prevent_double_submit' => true]);
+echo elgg_view_form('admin/security/settings', []);
```
Comment (jeabakker) [inline views/default/admin/security.php]: why leave the empty array?

**Review by jeabakker — CHANGES_REQUESTED** [review]

File: engine/hooks.php [hunk 7]
```diff
@@ -0,0 +1,55 @@
+<?php
```
[the remark this answers is not in the document]
Comment (jdalsem) [reply engine/hooks.php]: it is included in the similar way how we include actions and routes... so i assume so

[the remark this answers is not in the document]
Comment (jeabakker) [reply engine/hooks.php]: https://www.youtube.com/watch?v=7rr88Szc5q0

**Comment by jdalsem** [comment]
@jeabakker all requested changes implemented

File: engine/hooks.php (hunk 7 above)
[the remark this answers is not in the document]
Comment (jdalsem) [reply engine/hooks.php]: was expecting something like that... luckily we test...

**Review by jeabakker — APPROVED** [review]

**Pull request merged**